### PR TITLE
Fix Timer error on startup

### PR DESCRIPTION
**Pull request by KoBeWi**
Fixes https://github.com/godotengine/godot/pull/87542#discussion_r1888971859

### CODE REVIEW

**Comment by Mickeon** [comment]
It does fix the error, but it's a bit hacky, as before. It's also a bit inconsistent with the `set_warning_count(0);` below which may make this harder to follow. As I mentioned once, perhaps it's better to just bite the bullet and fetch the editor setting in the constructor...?

**Comment by KoBeWi** [comment]
Yeah that's an alternative.

**Comment by KoBeWi** [comment]
Pushed a different fix.

**Review by akien-mga — APPROVED** [review]

File: editor/code_editor.cpp [hunk 1]
```diff
@@ -1604,6 +1600,12 @@ void CodeTextEditor::_error_pressed(const Ref<InputEvent> &p_event) {
 
 void CodeTextEditor::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_READY: {
+			idle->set_wait_time(idle_time);
```
Comment (Mickeon) [inline editor/code_editor.cpp:1604]: `set_error_count` does this
https://github.com/godotengine/godot/blob/2b7ea6223bd0bdaa75706849c9dee1a82b2e2d01/editor/code_editor.cpp#L1629-L1638

So, one way or the other, the wait time will be set appropriately, which makes this line unnecessary. Am I missing something?

**Review by Mickeon — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**